collectors/REFERENCE.md

```diff
@@ -96,6 +96,7 @@ This section features a list of Netdata's plugins, with a boolean setting to ena
 	# PYTHONPATH environment variable = 
 	# proc = yes
 	# diskspace = yes
+    # timex = yes
```

Comment:
Please, take a look in the tabulation here,

Reply:
Fixed